spec/services/feeds/import_spec.rb

```diff
@@ -209,4 +209,25 @@
       expect(body_markdown).to include(expected_image_markdown)
     end
   end
+
+  context "when multiple users fetch from the same feed_url" do
+    it "fetches the articles in both accounts (if feed_mark_canonical = false)" do
+      rss_feed_user1 = Users::Setting.find_by(feed_url: link).user
+      rss_feed_user2 = create(:user)
+      rss_feed_user2.setting.update(feed_url: link)
+      expect { described_class.call(users: User.where(id: Users::Setting.where(feed_url: link).select(:user_id))) }
+        .to change(rss_feed_user1.articles, :count).by(10)
+        .and change(rss_feed_user2.articles, :count).by(10)
+    end
+
+    it "fetches the articles in both accounts (if feed_mark_canonical = true)" do
+      rss_feed_user1 = Users::Setting.find_by(feed_url: link).user
+      rss_feed_user1.setting.update(feed_mark_canonical: true)
+      rss_feed_user2 = create(:user)
+      rss_feed_user2.setting.update(feed_url: link, feed_mark_canonical: true)
```

Comment:
```suggestion
      rss_feed_user2.setting.update!(feed_url: link, feed_mark_canonical: true)
```

Reply:
Thanks for the insight @jgaskins ! As you mentioned, there are other specs in the file that don't use update!. Was wondering if I should change those. Better yet, we could even create a good first issue to do this for other specs!

Reply:
@squarebat Excellent idea!